genesyscloud/resource_genesyscloud_routing_skill_group.go

```diff
@@ -0,0 +1,379 @@
+package genesyscloud
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v72/platformclientv2"
+	"github.com/mypurecloud/terraform-provider-genesyscloud/genesyscloud/consistency_checker"
+)
+
+type SkillGroupsRequest struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+	Division    struct {
+		ID   string `json:"id,omitempty"`
+		Name string `json:"name,omitempty"`
+	} `json:"division,omitempty"`
+	SkillConditions struct{} `json:"skillConditions"` //Keep this here.  Even though we do not use this field in the struct The generated attributed is used as a placeholder
+}
+
+type AllSkillGroups struct {
+	Entities []struct {
+		ID   string `json:"id"`
+		Name string `json:"name"`
+	}
+	NextURI     string `json:"nextUri"`
+	SelfURI     string `json:"selfUri"`
+	PreviousURI string `json:"previousUri"`
+}
+
+//TODO
```

Comment:
These comments like TODO and //Done can be removed

Reply:
Done